### ISSUE TITLE
Show full PHP function definitions

### ISSUE THREAD

**Issue by borlak** [felixfbecker/php-language-server#558]
There is no return type, or argument information.  I have php.suggest.basic set to false, so as far as I can tell this is all coming from the intellisense extension.

Example of what I see (sorry, this is a camera pic):
![screen shot 2017-12-21 at 12 36 58 pm](https://user-images.githubusercontent.com/1967460/34274027-aef976b6-e64b-11e7-8d7d-f3c150afe4d0.png)

There is also the <?php but I think other issues have brought this up.

Example of what is desired:
<img width="500" alt="screen shot 2017-12-21 at 11 08 14 am" src="https://user-images.githubusercontent.com/1967460/34274039-bd3cabd0-e64b-11e7-8faa-7779ad129bfc.png">
This is from Netbeans, and there is a scrollbar which shows some more information about the return variable in this case.

**Comment by felixfbecker** [comment]
Yes, hovers only show the declaration line and description atm. Feel free to do a PR to improve

**Comment by phil-nelson** [comment]
This is actually very easy if we want to utilise the SignatureInformation that is now stored in definitions:

![function-hover](https://user-images.githubusercontent.com/8910254/34343246-8d7e4690-ea1c-11e7-8ac1-c866413d9a07.gif)

Looks a bit messy though, not sure if there's a better way to present this information

**Comment by felixfbecker** [comment]
Oh yeah, it's not a hard feature by any means. But it should definitely not be shown as a code block, but in some markdown table or list. I think it would also be good to take a general approach of not showing what's in the SignatureHelp, but showing a table of all `@tags` in the docblock.
